openapi.yml

```diff
@@ -546,15 +546,56 @@ components:
         view:
           type: string
           example: FACE
-          description: The view describes the "close-up" - primary foci, subject, or inner focus of the image. In practise this is the interior foci of what is captured in the 'orientation' field.
+          enum: [FACE,TAT,OTH,SCAR,MARK,OIC]
+          description: >
+            View is the subject focus; describing the inner focus or subject of the image, normally referring to a marking, tattoo or deeper level focus of the orientation. In practise this is the interior foci of what is captured in the 'orientation' field.
+              Options include: 
+              * `FACE` - Facing, 
+              * `TAT` - Tattoo, 
+              * `OTH` - Other, 
+              * `SCAR` - Scar, 
+              * `MARK` - Mark, 
+              * `OIC` - Offence in Custody
         orientation:
           type: string
           example: FRONT
-          description: The orientation describes the secondary foci, scope, or outer focus of the image, which is to be the anatomical body part or angle of an image. In practise this is the exterior foci of what is captured in the 'view' field.
+          enum: [ANKLE, ARM, DAMAGE, EAR, ELBOW, FACE, FIGHT, FINGER, FOOT, FRONT, HAND, HEAD, INCIDENT, INJURY, KNEE, LEG, LIP, NECK, NOSE, SHOULDER, THIGH, TOE, TORSO]
+          description: >
+            Orientation is the scope focus; describing the scope or outer focus of the image, normally referring to the highest level object of interest within the bounds of the photo itself. This is normally a body part or an angle of photography, such as a photo of someone’s FRONT or ARM.
+              Options include:
+              * `ANKLE` - Ankle,
+              * `ARM` - Arm,
+              * `DAMAGE` - Damage,
+              * `EAR` - Ear,
+              * `ELBOW` - Elbow,
+              * `FACE` - Face,
+              * `FIGHT` - Fight,
+              * `FINGER` - Finger,
+              * `FOOT` - Foot,
+              * `FRONT` - Front Facial View,
+              * `HAND` - Hand,
+              * `HEAD` - Head,
+              * `INCIDENT` - Incident,
+              * `INJURY` - Injury,
+              * `KNEE` - Knee,
+              * `LEG` - Leg,
+              * `LIP` - Lip,
+              * `NECK` - Neck,
+              * `NOSE` - Nose,
+              * `SHOULDER` - Shoulder,
+              * `THIGH` - Thigh,
+              * `TOE` - Toe,
+              * `TORSO` - Torso
         type:
```

Comment:
We're missing the `enum` property here! 👀

Reply:
Oops!